charts/frontend/templates/backup-cron.yaml

```diff
@@ -140,6 +140,7 @@ spec:
           {{- end }}
           {{- end }}
           restartPolicy: Never
+          serviceAccountName: {{ include "frontend.serviceAccountName" . }}
```

Comment:
Why does this not replace line 171 (imagepullsecrets) like in the drupal-project-k8s PR?

Reply:
fixed now, moved tests to a separate test suite